HerPortal.Data/HerDbContext.cs

```diff
@@ -33,10 +32,16 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             .Entity<CsvFileDownload>()
             .HasKey(cf => new
             {
-                cf.CustodianCode,
-                cf.Year,
-                cf.Month,
+                cf.CsvFileId,
                 cf.UserId,
+                cf.Timestamp,
+            });
+        
+        modelBuilder
+            .Entity<TrackedCsvFile>()
+            .HasKey(tcf => new
+            {
+                tcf.Id
             });
```

Comment:
I'm surprised you haven't had timezone issues with this. In the portal code we have to add:
```
        modelBuilder.Entity<CsvFileDownload>()
            .Property(cfd => cfd.TimeStamp)
            .HasColumnType("timestamp without time zone");
```
Is the portal using the same version of the Postgres library as the public site?